staging/src/github.com/kubeedge/mapper-framework/_template/mapper/Dockerfile_stream

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.20.10-bullseye
+
+WORKDIR /kubeedge
+
+ENV GO111MODULE=on \
+    GOPROXY=https://goproxy.cn,direct
+
+COPY . .
+
+RUN apt-get update && \
+    apt-get install -y bzip2 upx-ucl gcc-aarch64-linux-gnu libc6-dev-arm64-cross gcc-arm-linux-gnueabi libc6-dev-armel-cross libva-dev libva-drm2 libx11-dev libvdpau-dev libxext-dev libsdl1.2-dev libxcb1-dev libxau-dev libxdmcp-dev yasm
+
+RUN curl -sLO https://ffmpeg.org/releases/ffmpeg-4.1.6.tar.bz2 && \
+    tar -jx --strip-components=1 -f ffmpeg-4.1.6.tar.bz2 &&  \
+    ./configure &&  make && \
+    make install
+
+RUN GOOS=linux go build -o main cmd/main.go
```

Comment:
Is it necessary to separate into two dockerfile? also cc @WillardHu

Reply:
Since the stream mapper needs to download additional dependencies, it takes about five minutes to build the image, so I feel it would be better to separate it without affecting the construction of other mapper images.